content/en/serverless/azure_app_services/azure_app_services_container.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Azure App Service - Linux Container
+title: Instrument Azure App Service - Linux Containers
```

Comment:
```suggestion
title: Instrument Azure App Service - Linux Container with serverless-init
```

Reply:
I'd actually like to leave this title as-is. This page should basically be the default recommended install, which is serverless-init, but should become sidecar in the future)

Reply:
Okay, fair enough :)